packages/p2p/src/components/advertiser-page/advertiser-page.jsx

```diff
@@ -100,26 +100,26 @@ const AdvertiserPage = () => {
                                             is_readonly
                                             number_of_stars={5}
                                             should_allow_hover_effect={false}
-                                            star_size={20}
+                                            star_size={isMobile() ? 17 : 20}
                                         />
-                                        &nbsp;
-                                        <Text color='prominent' size={isMobile() ? 'xxxs' : 'xs'}>
-                                            {rating_average_decimal}
-                                        </Text>
-                                        &nbsp;
-                                        <Text color='less-prominent' size={isMobile() ? 'xxxs' : 'xs'}>
-                                            {rating_count === 1 ? (
-                                                <Localize
-                                                    i18n_default_text='({{number_of_ratings}} rating)'
-                                                    values={{ number_of_ratings: rating_count }}
-                                                />
-                                            ) : (
-                                                <Localize
-                                                    i18n_default_text='({{number_of_ratings}} ratings)'
-                                                    values={{ number_of_ratings: rating_count }}
-                                                />
-                                            )}
-                                        </Text>
+                                        <div className='advertiser-page__rating--text'>
```

Comment:
Aren't these changes from `my-profile-rating` branch? Are they required here? Can we limit the scope of this branch to only include flow for order ratings?

Reply:
All I changed is that I removed the now unnecessary `&nbsp;`, just cleaning up

Reply:
Not just this file, but the changes in `packages/p2p/src/components/my-profile/my-profile-stats/my-profile-name/my-profile-name.jsx` and it's scss etc as well. I just didn't want to put the same comment multiple times. Are those changes required in this branch?

Reply:
For example, `RecommendedBy` component is used in `MyProfileName` which is supposed to be the changes in `my-profile-rating` branch. So, are these changes required in this branch?

Reply:
Yes, it's also used here.